third_party/http-parser/README.md

```diff
@@ -0,0 +1,27 @@
+# http-parser
+
+## URL
+
+https://github.com/nodejs/http-parser/releases
+
+## Version 
+
+2.9.4
+
+## Commit
+
+2343fd6b5214b2ded2cdcf76de2bf60903bb90cd
```

Comment:
It would be better if this commit hash can be a link to the release we are using.

Reply:
Sound good!